### DISCUSSION TITLE
GitHub Actions does not support clickable links in the annotations directly

### DISCUSSION THREAD

**Discussion by FrancescaRodricks** [community/community#118819] [Actions]
### Select Topic Area

Product Feedback

### Body

For example 
```
- name: Create annotation with a link
  run: echo "::error file=app.js,line=1,col=1::Something went wrong. Check the documentation at https://example.com."
```

I would expect to see  "https://example.com" as a link rather than plain text in the workflow log. Users have to copy this URL and paste it into their browser to access the documentation. 

Will this be fixed anytime soon?

**Comment by septatrix** [comment]
Maybe OSC 8 (an ANSI escape sequence) could be used for detecting those

https://github.com/Alhadis/OSC8-Adoption
